src/hoa-renderer.js

```diff
@@ -91,41 +91,34 @@ HOARenderer.prototype._initializeCallback = function(resolve, reject) {
   // This is because Chrome cannot decode the audio file >8  channels.
   var audioBufferData = [];
   this._HRIRUrls.forEach(function(key, index, urls) {
-    audioBufferData.push({name: key, url: urls[index]});
+    audioBufferData.push({name: index, url: urls[index]});
```

Comment:
Yes, this is confusing. We need to use indexes so that we know the order of the files and hence what channels to copy each file into.